runhouse/servers/http/http_server.py

```diff
@@ -612,6 +624,92 @@ async def rename_object(request: Request, params: RenameObjectParams):
                 e, traceback.format_exc(), from_http_server=True
             )
 
+    @staticmethod
+    @app.post("/folder/method/ls")
```

Comment:
maybe just `/folder/ls`?

Reply:
this conflicts with the `@app.post("/{key}/{method_name}")` method, but welcome to other ideas for the naming convention we use for these endpoints